src/cmd/root.go

```diff
@@ -36,16 +36,21 @@ func init() {
 	rootCmd.PersistentFlags().String("api-token", "", "The OpsLevel API Token. Overrides environment variable 'OPSLEVEL_API_TOKEN'")
 	rootCmd.PersistentFlags().String("log-format", "TEXT", "overrides environment variable 'OPSLEVEL_LOG_FORMAT' (options [\"JSON\", \"TEXT\"])")
 	rootCmd.PersistentFlags().String("log-level", "INFO", "overrides environment variable 'OPSLEVEL_LOG_LEVEL' (options [\"ERROR\", \"WARN\", \"INFO\", \"DEBUG\"])")
+	rootCmd.PersistentFlags().Bool("scaling-enabled", false, "Enables built-in pod scaling for kubernetes environment, defaults to false for local development")
 
+	// TODO: Update pod-* flags to job-pod-* to distinguish between job and runner specific values.  Moving this work over to separate ticket since it's a breaking change: https://gitlab.com/jklabsinc/OpsLevel/-/issues/5763
 	rootCmd.PersistentFlags().Int("pod-max-wait", 60, "The max amount of time to wait for the job pod to become healthy.")
-	rootCmd.PersistentFlags().String("pod-namespace", "default", "The kubernetes namespace to create pods in.")
+	rootCmd.PersistentFlags().String("pod-namespace", "default", "The kubernetes namespace to create job pods in.")
 	rootCmd.PersistentFlags().Int64("pod-requests-cpu", 1000, "Default is in millicores.")
 	rootCmd.PersistentFlags().Int64("pod-requests-memory", 1024, "Pod job resource requests in MB.")
 	rootCmd.PersistentFlags().Int64("pod-limits-cpu", 1000, "Default is in millicores.")
 	rootCmd.PersistentFlags().Int64("pod-limits-memory", 1024, "Pod job resource limits in MB.")
 	rootCmd.PersistentFlags().String("pod-shell", "/bin/sh", "The shell to use for commands inside the pod.")
 	rootCmd.PersistentFlags().Int("pod-log-max-interval", 30, "The max amount of time between when pod logs are shipped to OpsLevel. Works in tandem with 'pod-log-max-size'")
 	rootCmd.PersistentFlags().Int("pod-log-max-size", 1000000, "The max amount in bytes to buffer before pod logs are shipped to OpsLevel. Works in tandem with 'pod-log-max-interval'")
+	rootCmd.PersistentFlags().String("runner-pod-name", "", "overrides environment variable 'RUNNER_POD_NAME'")
```

Comment:
might want to call out this needs to be unique per pod - so its best to use the downward API of k8s

Reply:
Yeah, I was thinking this would mostly be used for local testing which is what I've been using it for but I was planning on setting this in the manifest like this:  
```
spec:
  template:
    spec:
      containers:
        - args:
            - "-node-id=$(POD_NAME)"
          env:
            - name: POD_NAME
              valueFrom:
                fieldRef:
                  apiVersion: v1
                  fieldPath: metadata.name
```

Reply:
👍 could even just do
```yaml
Yeah, I was thinking this would mostly be used for local testing which is what I've been using it for but I was planning on setting this in the manifest like this:

spec:
  template:
    spec:
      containers:
          env:
            - name: RUNNER_POD_NAME
              valueFrom:
                fieldRef:
                  apiVersion: v1
                  fieldPath: metadata.name